Modules/System/Cryptography Management/src/CryptographyManagementImpl.Codeunit.al

```diff
@@ -610,4 +609,90 @@ codeunit 1279 "Cryptography Management Impl."
             Error(GetLastErrorText());
         Hash := Convert.ToBase64String(ByteArray);
     end;
+
+    procedure EncryptTextWithDESCryptoServiceProvider(VarInput: Text; Password: Text; Salt: Text) VarOutput: Text
```

Comment:
The name of the module already contains key word DES, so let's make it simple - either Encrypt or EncryptText only